src/main/python/systemds/context/systemds_context.py

```diff
@@ -425,41 +425,6 @@ def full(self, shape: Tuple[int, int], value: Union[float, int]) -> 'Matrix':
         named_input_nodes = {'rows': shape[0], 'cols': shape[1]}
         return Matrix(self, 'matrix', unnamed_input_nodes, named_input_nodes)
 
-
-    def fft(self, real_input: 'Matrix') -> 'MultiReturn':
-        """
-        Performs the Fast Fourier Transform (FFT) on the matrix.
-        :param real_input: The real part of the input matrix.
-        :return: A MultiReturn object representing the real and imaginary parts of the FFT output.
-        """
-
-        real_output = OperationNode(self, '', output_type=OutputType.MATRIX, is_python_local_data=False)
-        imag_output = OperationNode(self, '', output_type=OutputType.MATRIX, is_python_local_data=False)
-
-        fft_node = MultiReturn(self, 'fft', [real_output, imag_output], [real_input])
-
-        return fft_node
-
-
-    def ifft(self, real_input: 'Matrix', imag_input: 'Matrix' = None) -> 'MultiReturn':
-        """
-        Performs the Inverse Fast Fourier Transform (IFFT) on a complex matrix.
-        
-        :param real_input: The real part of the input matrix.
-        :param imag_input: The imaginary part of the input matrix (optional).
-        :return: A MultiReturn object representing the real and imaginary parts of the IFFT output.
-        """
-
-        real_output = OperationNode(self, '', output_type=OutputType.MATRIX, is_python_local_data=False)
-        imag_output = OperationNode(self, '', output_type=OutputType.MATRIX, is_python_local_data=False)
-
-        if imag_input is not None:
-            ifft_node = MultiReturn(self, 'ifft', [real_output, imag_output], [real_input, imag_input])
-        else:
-            ifft_node = MultiReturn(self, 'ifft', [real_output, imag_output], [real_input])
-
-        return ifft_node
-
```

Comment:
moved to matrix.py, cuz of the indirect creation of the Matrix-OperationNode (which was done to avoid cyclic dependency)